Lib/unittest/mock.py

```diff
@@ -737,14 +737,47 @@ def _format_mock_failure_message(self, args, kwargs):
         return message % (expected_string, actual_string)
 
 
+    def _get_call_signature_from_name(self, name):
+        """
+        If there is no name like the case for call check against functions
+        with tuples then return the self's spec signature.
```

Comment:
I can't really follow what this sentence is saying, could it be reworded?

Reply:
I have reworded the comment. Feel free to suggest if it can be rephrased better.

* If an assertion is made against a method like `obj.meth1.assert_has_calls([call(1)]` here `call(1)` object has no name to lookup and hence just return `obj.meth1._spec_signature`. 
* If an assertion is made against the obj like `obj.assert_has_calls([call.meth1(1)])` here `call.meth1(1)` will have name 'meth1' that should be used to return signature of `obj.meth1` unlike the bug where `obj` signature was returned.

Thanks